### PR TITLE
Edge specs

### PR DESCRIPTION
**Pull request by wimpers**
@khenderick @saelbrec please review

### CODE REVIEW

File: specs/Edge.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Design choices
+- No Edge manager is required
+ - This means we can't list all disks connected to certain host without retrieving the whole list
+- If possible don't model the edge
+
+
+## Required changes
+- Implement storage and management IP correctly
+    - DTL and MDS should use the management Ip
```
Comment (khenderick) [inline specs/Edge.md]: Shouldn't they also use the storage ip?

Comment (wimpers) [inline specs/Edge.md]: I was under the impression this was mgmt ip? @saelbrec  can you shed  a light?

Comment (saelbrec) [inline specs/Edge.md]: Best practice is to configure storage ip on the network interface with the lowest latency, which is what you want for DTL & MDS.

**Comment by wimpers** [comment]
Updated the spec. Can you verify if the spec is now correct?

**Comment by khenderick** [comment]
Looks good to me. However, for some items the actual implementation will depend on the available information.

**Comment by saelbrec** [comment]
@redlicha , @cnanakos ,
Please check the specs on what is required in the GUI and then decide on how this can be retrieved.
We discussed this in the past and I thought there was a consensus to be able to retrieve this via the volumedriver XMLRPC API. Most likely included in the volume.info ?
Whatever needs to be done from your end can be done under the umbrella of this story.

**Pull request merged**